.github/workflows/test-lightwalletd.yml

```diff
@@ -0,0 +1,370 @@
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd suite
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_PATH -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_test_suite
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+
+  # TODO: add description to this test and verify name
+  test-fully_synced_rpc:
+    name: Test with a getblock RPC call
+    runs-on: ubuntu-latest
+    needs: [build]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@v3.0.2
+        with:
+          persist-credentials: false
+
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Downcase network name for disks
+        run: |
+          NETWORK_CAPS=${{ github.event.inputs.network || env.NETWORK }}
+          echo "NETWORK=${NETWORK_CAPS,,}" >> $GITHUB_ENV
+
+      # Setup gcloud CLI
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v0.7.1
+        with:
+          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
+          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
+          token_format: 'access_token'
+
+      # Check if our destination compute instance exists and delete it
+      - name: Delete existing instance with same SHA
+        run: |
+          INSTANCE=$(gcloud compute instances list --filter=sync-lwd-rpc-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} --format='value(NAME)')
+          if [ -z "${INSTANCE}" ]; then
+            echo "No instance to delete"
+          else
+            gcloud compute instances delete "${INSTANCE}" --zone "${{ env.ZONE }}" --delete-disks all --quiet
+          fi
+
+      # Creates Compute Engine virtual machine instance w/ disks
+      - name: Create GCP compute instance
+        run: |
+          gcloud compute instances create-with-container "sync-lwd-rpc-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}" \
+          --boot-disk-size 100GB \
+          --boot-disk-type pd-ssd \
+          --container-image ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} \
+          --container-restart-policy=never \
+          --container-stdin \
+          --container-tty \
+          --container-env=ZEBRA_SKIP_IPV6_TESTS=1,TEST_LWD_RPC_SYNC=1,ZEBRA_FORCE_USE_COLOR=1 \
```

Comment:
Question: Where is the test name for this job?

Reply:
I'm using `TEST_LWD_RPC_SYNC ` in the Dockerfile `entrypoin.sh`

As this formart to create containers does not allow to use `--` in a single line as an argument, as stated here:

cargo test --locked --release --features enable-sentry --test acceptance `--` --nocapture ...

This happened with other Full sync test before, which is also activated in GCP with the `TEST_FULL_SYNC ` variable.

Reply:
It might be helpful to mention `entrypoint.sh` in a comment.